packages/next-app/src/app/internalmanager/(components)/TokenSelect.tsx

```diff
@@ -10,23 +11,18 @@ import { useAccount, useNetwork } from "wagmi";
 
 import genericTokenLogo from "#/assets/generic-token-logo.png";
 import { Dialog } from "#/components/Dialog";
-import { Input } from "#/components/Input";
 import Table from "#/components/Table";
 import { impersonateWhetherDAO, internalBalances } from "#/lib/gql";
-import { moralisGetTokens } from "#/lib/moralis";
 import { refetchRequest } from "#/utils/fetcher";
 import { ArrElement, GetDeepProp } from "#/utils/getTypes";
 
-interface tokenItem {
-  token_address: string;
-  name: string;
-  symbol: string;
-  logo?: string | undefined;
-  thumbnail?: string | undefined;
-  decimals: number;
-  balance: string;
-  possible_spam: boolean;
-  internalBalance?: string;
+interface TokenWalletBalance extends FetchBalanceResult {
+  tokenAddress: string;
+}
+
+interface TokenItem extends TokenWalletBalance {
+  name?: string | null;
```

Comment:
why token name is now optional and nullable?

Reply:
it's how the type comes from the gql